scripts/scil_compute_connectivity.py

```diff
@@ -163,11 +155,11 @@ def _build_arg_parser():
     p = argparse.ArgumentParser(
         description=__doc__,
         formatter_class=argparse.RawTextHelpFormatter,)
-    p.add_argument('in_bundles_dir',
-                   help='Folder containing all the bundle files (.trk).')
-    p.add_argument('labels_list',
-                   help='Text file containing the list of labels from the '
-                        'atlas.')
+    p.add_argument('in_hdf5',
+                   help='Input filename for the hdf5 container (.h5).')
```

Comment:
Input filename for the hdf5 container coming from scil_decompose_connectivity (.h5) 

(if you want)